.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: continuous-integration
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    env:
+      ATHENS_MONGO_STORAGE_URL: mongodb://localhost:27017
+      ATHENS_MINIO_ENDPOINT: localhost:9000
+      REDIS_TEST_ENDPOINT: localhost:6379
+      REDIS_SENTINEL_TEST_ENDPOINT: localhost:26379
+      REDIS_SENTINEL_TEST_MASTER_NAME: redis-1
+      REDIS_SENTINEL_TEST_PASSWORD: sekret
+      PROTECTED_REDIS_TEST_ENDPOINT: localhost:6380
+      ATHENS_PROTECTED_REDIS_PASSWORD: AthensPass1
+    runs-on: ubuntu-latest
+    services:
+      mongo:
+        image: mongo
+        ports:
+          - 27017:27017
+      minio:
+        image: minio/minio:edge-cicd
```

Comment:
Can we use minio:latest? Or why this image

Reply:
`minio:latest` doesn't start the server by default and there's no option to specify command/ args while starting a service in github actions hence `latest` can't be used.

`minio/minio:edge-cicd` was added for this specific purpose: https://github.com/minio/minio/pull/10933.